test/tests/functional/pbs_job_array.py

```diff
@@ -581,7 +581,7 @@ def test_delete_history_subjob_server_restart(self):
         self.server.manager(MGR_CMD_SET, SERVER, a)
         a = {'resources_available.ncpus': 2}
         self.server.manager(MGR_CMD_SET, NODE, a, self.mom.shortname)
-        j = Job(TEST_USER, attrs={
+        j = Job(PBSROOT_USER, attrs={
```

Comment:
How does submitting the job as PBSROOT_USER instead of TEST_USER add -koe?  Shouldn't you just add -koe to the attrs?